src/taskgraph/run-task/run-task

```diff
@@ -1101,8 +1048,7 @@ def main(args):
         add_vcs_arguments(parser, repository, name)
 
     parser.add_argument('--fetch-hgfingerprint', action='store_true',
-                        help='Fetch the latest hgfingerprint from the secrets store, '
-                        'using the taskclsuerProxy')
+                        help="Obsolete option, ignored for backwards compatibility")
```

Comment:
Might be better to set `help=argparse.SUPPRESS`. That way the option won't even show up in the help in the first place.

Reply:
Thanks for the tip :)